apps/emqx_bridge_mongodb/src/emqx_bridge_mongodb_connector.erl

```diff
@@ -61,13 +61,8 @@ on_get_channel_status(InstanceId, _ChannelId, State) ->
 on_get_channels(InstanceId) ->
     emqx_bridge_v2:get_channels_for_connector(InstanceId).
 
-on_get_status(InstanceId, ConnectorState = #{connector_state := DriverState0}) ->
-    case emqx_mongodb:on_get_status(InstanceId, DriverState0) of
-        {Status, DriverState, Reason} ->
-            {Status, ConnectorState#{connector_state := DriverState}, Reason};
```

Comment:
What's the original purpose of this state mutation here ?

Reply:
I can't answer for the original author, but looking at the code, we can see it's accommodating for the possibility of the state changing for whatever reason, given the previous signature for the `emqx_resource:on_get_status` callback.  Which is changed here in this PR.

Reply:
ok, so it means this state is never mutated for mongodb anyway. which is good.
resolving this comment.